Containerfile

```diff
@@ -1,47 +1,64 @@
-#syntax=docker/dockerfile-upstream:1.4.0-rc1
-ARG RUSTUP_TOOLCHAIN
-FROM docker.io/library/rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-build
-RUN apt-get update &&\
-    apt-get install -y curl
+#syntax=docker/dockerfile-upstream:1.4
+
 
-RUN cargo install cargo-chef
+# Base image for builds and cache
+ARG RUSTUP_TOOLCHAIN
+FROM rust:${RUSTUP_TOOLCHAIN}-buster as shuttle-build
+RUN apt update && apt install -y curl
```

Comment:
Do we still need curl here btw? :thinking:

Reply:
Moved to deployer. Only used there to install protoc.

Reply:
Cool, thanks!